task_functions.py

```diff
@@ -1377,33 +852,249 @@ def fill_elastic_index(**kwargs):
+                        s.code_postal as code_postal,
+                        s.commune as commune,
+                        s.commune_2 as commune_2,
+                        s.complement_adresse as complement_adresse,
+                        s.complement_adresse_2 as complement_adresse_2,
+                        s.date_creation as date_creation,
+                        s.date_debut_activite as date_debut_activite,
+                        s.distribution_speciale as distribution_speciale,
+                        s.distribution_speciale_2 as distribution_speciale_2,
+                        s.enseigne_1 as enseigne_1,
+                        s.enseigne_2 as enseigne_2,
+                        s.enseigne_3 as enseigne_3,
+                        s.est_siege as est_siege,
+                        s.etat_administratif_etablissement as
+                        etat_administratif_etablissement,
+                        s.geo_adresse as geo_adresse,
+                        s.geo_id as geo_id,
+                        s.indice_repetition as indice_repetition,
+                        s.indice_repetition_2 as indice_repetition_2,
+                        s.latitude as latitude,
+                        s.libelle_cedex as libelle_cedex,
+                        s.libelle_cedex_2 as libelle_cedex_2,
+                        s.libelle_commune as libelle_commune,
+                        s.libelle_commune_2 as libelle_commune_2,
+                        s.libelle_commune_etranger as libelle_commune_etranger,
+                        s.libelle_commune_etranger_2 as libelle_commune_etranger_2,
+                        s.libelle_pays_etranger as libelle_pays_etranger,
+                        s.libelle_pays_etranger_2 as libelle_pays_etranger_2,
+                        s.libelle_voie as libelle_voie,
+                        s.libelle_voie_2 as libelle_voie_2,
+                        (SELECT liste_finess FROM finess WHERE siret = s.siret) as
+                        liste_finess,
+                        (SELECT liste_idcc FROM convention_collective WHERE siret =
+                        s.siret) as liste_idcc,
+                        (SELECT liste_rge FROM rge WHERE siret = s.siret) as liste_rge,
+                        (SELECT liste_uai FROM uai WHERE siret = s.siret) as liste_uai,
+                        s.longitude as longitude,
+                        s.nom_commercial as nom_commercial,
+                        s.numero_voie as numero_voie,
+                        s.numero_voie_2 as numero_voie_2,
+                        s.siren as siren,
+                        s.siret as siret,
+                        s.tranche_effectif_salarie as tranche_effectif_salarie,
+                        s.type_voie as type_voie,
+                        s.type_voie_2 as type_voie_2
+                        FROM siret s
+                        WHERE s.siren = ul.siren
+                    )
+                ) as etablissements,
```

Comment:
Too bad there is no nested fields in this nested field, that's not so challenging in fact. But I think you are now master in sql.

Reply:
![image](https://user-images.githubusercontent.com/48837850/210834097-f8b1c168-ee75-4ec6-b0c7-1a0090606ce2.png)